cmd/blockchaincmd/import_public.go

```diff
@@ -221,17 +222,29 @@ func importBlockchain(
 		sc.Networks[network.Name()] = e
 		printFunc("  Validator Manager Address: %s", validatorManagerAddress)
 		if rpcURL != "" {
-			sc.ValidatorManagement, err = validatorManagerSDK.GetValidatorManagerType(rpcURL, common.HexToAddress(validatorManagerAddress))
-			if err != nil {
-				return models.Sidecar{}, nil, fmt.Errorf("could not obtain validator manager type: %w", err)
+			sc.ValidatorManagement = validatorManagerSDK.GetValidatorManagerType(rpcURL, common.HexToAddress(validatorManagerAddress))
+			if sc.ValidatorManagement == validatormanagertypes.UndefinedValidatorManagement {
+				return models.Sidecar{}, nil, fmt.Errorf("could not infer validator manager type")
 			}
-			printFunc("  Validation Kind: %s", sc.ValidatorManagement)
 			if sc.ValidatorManagement == validatormanagertypes.ProofOfAuthority {
 				owner, err := contract.GetContractOwner(rpcURL, common.HexToAddress(validatorManagerAddress))
 				if err != nil {
 					return models.Sidecar{}, nil, err
 				}
-				sc.ValidatorManagerOwner = owner.String()
+				// check if the owner is a specialized validator manager
+				validatorManagement := validatorManagerSDK.GetValidatorManagerType(rpcURL, owner)
```

Comment:
Can you provide example of how we could get to this point? Don't really understand it

Reply:
added a explanatory comment